lib/crypto/crypto_compat.c

```diff
@@ -158,7 +158,7 @@ crypto_compat_RSA_generate_key()
 {
 	RSA * key;
 
-#if OPENSSL_VERSION_NUMBER < 0x10100000L
+#if OPENSSL_VERSION_NUMBER < 0x00908000L
```

Comment:
I _think_ that the OS X warnings are from the (then-upcoming) complete deprecation, rather than this specific function.